### DISCUSSION TITLE
How to edit facial expressions without armpits changing

### DISCUSSION THREAD

**Discussion by sethjames9001** [soupday/cc3_blender_tools#289] [Q&A]
I am attempting to edit facial expressions but the armpits are being altered when I send the mesh back to CC4. This seems to be an issue that was re introduced with the latest updates. I have succesfully edited facial expressions before but the method is no longer working.
On the previous versions of the add on I would open the facial expression editor in CC4 and click send pose to Blender. This would put the character into a similar T Pose in Blender. I could then edit the shape keys within Blender and send them back to CC4 for updating. Since updating to version 2.2 the armpits are getting altered. Also when sending a pose, it breaks driver for the jaw bone. Jaw Open will open the mouth but after sending any pose from CC4 the jaw bone is no longer drivin by the shapekey in Blender.

**Comment by soupday** [comment]
The problem happening here is that the T-Pose is **not** the bind pose of the model. The way Blender and CC/iClone deform the mesh around the bones is very slightly different. The character needs to be in the bind pose (rest position in Blender) so that neither side is doing _any_ skinning deformation on the mesh.

You can either:
- In Blender put the armature in _rest position_, then send that pose back to CC4.
or
- In CC4 you can use the **Remove** > **Restore Bind Pose** function in the Animation Player in CC4 and send that pose to Blender.

I need to fix the shape-key drivers breaking from the pose functions, but for now the drivers can be rebuilt from the **Character Build Settings panel** > **Rebuild Drivers** button.

(It's a simple enough fix, pushed to [Dev branch](https://github.com/soupday/cc_blender_tools/tree/Dev) if you want to try it)

**Comment by sethjames9001** [comment]
Wow Thank you Victor!!!

I was unaware of being able to return to bind pose while still in facial edit mode! I thought the character had to be in T Pose while editing the facial shapes.

I think what I was doing a few months ago... I was sending the T Pose mesh from Blender back every time before an edit. So the armpits would be similar before and after the edit. I remember now though that each time the armpits would change a little. So the original mesh was getting changed a little bit each time resulting in caved in armpits. I fixed the armpits afterwards.

Anyway, restoring bind pose after entering the facial editor seems to have solved my issues.

Thank you soooo much for your quick response and for your development of this add on.

I really had given up on even using Character Creator until I discovered the Go B addon and being able to quickly go back and forth from Blender for edits.

"rebuild Drivers" I also was unaware of that! It really helps! Thanks again!





________________________________
